### DISCUSSION TITLE
Payment issues in India

### DISCUSSION THREAD

**Discussion by thefoodiecoder** [community/community#67932] [New to GitHub]
I have been trying to make payment on Github but it never goes through. 

Initially I was adding 11 seats that **took me 3 days** to add after I used one of my employees credit card that worked. Other cards and paypal kept failing. 

Now after a few days I am adding few more team members but no luck with same card and other **Debit Cards, Credit Cards and Paypal**. These all are working on other platforms. 

Github's customer support takes multiple days to reply. At least payment queries should be addressed fast. We were planning a transition from Bitbucket for our whole ORG, but with such issues we are not able to proceed and whole plan is delayed now.

**Comment by queenofcorgis** [comment]
Hi @thefoodiecoder, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
